neuraxle/steps/util.py

```diff
@@ -230,3 +231,11 @@ def inverse_transform(self, data_output):
 
 class DataShuffler:
     pass  # TODO.
+
+
+class IdentityPipelineRepository(PipelineRepository):
```

Comment:
I like the sound of that class. Although I'm thinking of renaming a `PipelineRepository` more like a `Saver`, not sure if this is the idea I had in the previous commit's review above.